components/ProductMatrix.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+    <div v-if="error">{{ error }}</div>
+    <div class="product-matrix">
+      <div v-for="card in cards" :key="card.sortKey">
+        <component :is="components[card.type]" :product="card" />
+      </div>
+    </div>
+    <v-row align="center">
+      <v-col class="text-center" cols="12">
```

Comment:
@coopjakob I think we can abandon it. I just saw you used it (Vuetify) your initial app and I didn't realize this would be injected into another app. We should just use plain scss to style it like the original site